gatorgrouper/utils/display.py

```diff
@@ -21,10 +21,9 @@ def display_student_groups(student_groups):
 
 def display_welcome_message():
     """ Display a welcome message """
-    print()
-    print("GatorGrouper: Automatically Assign Students to Groups")
-    print("https://github.com/GatorGrouper/gatorgrouper")
-    print()
+    firstline = "\nGatorGrouper: Automatically Assign Students to Groups"
+    secondline = "\nhttps://github.com/GatorGrouper/gatorgrouper\n"
+    return print(firstline + secondline)
```

Comment:
`print` does not return anything -- in order to test this method, you need to capture `sys.out`, which is covered on this [page](https://docs.pytest.org/en/latest/capture.html#accessing-captured-output-from-a-test-function) of the `pytest` documentation.

Is there any reason why the code here has to change? There is no need to create string variables or anything like that, multiple print statements are perfectly fine. I would highly suggest using the original code and simply using the `capsys` fixture as described in the link above to capture the printed output.

Reply:
Thanks, we will work on this soon.

Reply:
This has been fixed by https://github.com/GatorEducator/gatorgrouper/pull/185/commits/eecc5d46a1fe67615f931db711e76a745cc2fc59